backend/create.php

```diff
@@ -0,0 +1,109 @@
+<?php 
+
+include 'config.php';
+
+// globals
+$queries = [];
+
+// error would be empty submissions
+function create_account($json_hash) {
+	GLOBAL $db;
+	$valid = true; //set to false if new account cannot be added
+	$name = "";
+	$password = "";
+
+	// list of fields that must not be null
+	$fields = [
+		"name",
+		"password"
+	];
+
+	// check validity of json
+	foreach($fields as $f) {
+		if(!$json_hash[$f]) {
+			print "Error: $f cannot be null <br>";
+			$valid = false;
+		}
+	}
+	if(!$valid) {
+		print "Query body has one or more errors <br>";
+		return;
+	}
+
+	$name = mysqli_real_escape_string($db, $json_hash['name']);
+	$password = mysqli_real_escape_string($db, $json_hash['password']);
+
+	array_push(
+		$GLOBALS['queries'],
+		"INSERT INTO Accounts(name,password) VALUES('$name', '$password')" //TODO cleanup redundancy with $fields loop
+	);
+}
+
+function create_task($json_hash){
+	GLOBAL $db;
+	$valid = true;
+	$account_id = "";
+	$tasks_name = "";
+	$category = "";
+    $deadline = "";
+    $priority = "";
+
+	// list of fields that must not be null
+	$fields = [
+		"account_id",
+		"tasks_name",
+		"category",
+    	"deadline",
+    	"priority"
```

Comment:
Should these also be indented?
```suggestion
    		"deadline",
    		"priority"
```